itk-event-database/event-database-api
# Feature/ev 239 update dev setup

https://jira.itkdev.dk/browse/EV-239

* Update to latest itk docker setup
* Support symfony cli
* Migrate from CircleCI (no longer worked) to Github Actions
* Update git commit hooks to use docker (needs work, fails in image)
* Update and apply coding standards to ensure check passes
